### PR TITLE
Issue #6301: fix docblock for function

### LINKED ISSUE #6301: backdrop_find_theme_functions doc should say it returns array not $implementations
Currently:

```
/**
 * Allows themes and/or theme engines to discover overridden theme functions.
 *
 * @param $cache
 *   The existing cache of theme hooks to test against.
 * @param $prefixes
 *   An array of prefixes to test, in reverse order of importance.
 *
 * @return $implementations
 *   The functions found, suitable for returning from hook_theme;
 */
function backdrop_find_theme_functions($cache, $prefixes) {
```

Should be

```
/**
 * Allows themes and/or theme engines to discover overridden theme functions.
 *
 * @param $cache
 *   The existing cache of theme hooks to test against.
 * @param $prefixes
 *   An array of prefixes to test, in reverse order of importance.
 *
 * @return array
 *   An array of the functions found, suitable for returning from hook_theme;
 */
function backdrop_find_theme_functions($cache, $prefixes) {
```

### PR DESCRIPTION
**Pull request by herbdool**
Fixes https://github.com/backdrop/backdrop-issues/issues/6301

### CODE REVIEW

**Comment by herbdool** [comment]
Random fails. Plus spelling that is hopefully being added elsewhere because it's not related to this.

File: core/includes/theme.inc [hunk 1]
```diff
@@ -1254,8 +1254,9 @@ function path_to_theme() {
  * @param $prefixes
  *   An array of prefixes to test, in reverse order of importance.
  *
- * @return $implementations
- *   The functions found, suitable for returning from hook_theme;
+ * @return array
+ *   An array of the functions found, suitable for returning from hook_theme.
```
Comment (bugfolder) [inline core/includes/theme.inc]:
```suggestion
 *   An array of the functions found, suitable for returning from hook_theme.
```

Remove extra whitespace.

Comment (avpaderno) [inline core/includes/theme.inc]: Should not *An array of the retrieved functions, suitable for being returned from hook_theme().* be better?

Comment (quicksketch) [reply core/includes/theme.inc]: I left this as-is so the description fits on one line.

**Comment by quicksketch** [comment]
Thanks folks! I merged into 1.x and 1.26.x.

**Pull request merged**